index.md

```diff
@@ -0,0 +1,31 @@
+This file lists the available Mattermost plugins.
+
+To keep it Bot-parseable, only plugin listings can include `[]()` formatted URL references.
+
```

Comment:
Maybe add a description for each plugin? This would make it easier to find the plugin you look for. The description can be just copy pasted from the manifest. 1/5
This could be done in a markdown table, which becomes more useful if we want to add more data for each plugin. 0/5

Reply:
Long-term plan is to include these in an integrations directory/marketplace. We're pretty close to releasing V1 on our web, aim is by end of Marh. They will include a longer description.

So 0/5 if worth adding here.

Reply:
I was hesitant (what's the authoritative source, the manifest?), can do this later as a separate PR